src/plugins/AngularPlugin.ts

```diff
@@ -20,14 +20,35 @@ export default class AngularPlugin implements ConfigPlugin {
             {
               test: tsRule.test,
               use: { loader: 'angular2-template-loader', options: spin.createConfig(builder, 'angular2Template', {}) }
+            },
+            /**
+             * Enabling the SystemJS parser for the .js files under @angular/core/ will disable
+             * the deprecation warnings.
+             * See https://prntscr.com/lezfmd
+             *
+             * SystemJS is disabled by default for webpack.
+             * See https://webpack.js.org/configuration/module/#rule-parser
+             *
+             * Note that the webpack.ContextReplacementPlugin must also be properly configured (see below).
+             */
+            {
+              test: /[\\\/]@angular[\\\/]core[\\\/].+\.js$/,
+              parser: { system: true }
             }
           ]
         },
         plugins: [
           // Workaround for angular/angular#11580
           new webpack.ContextReplacementPlugin(
-            // The (\\|\/) piece accounts for path separators in *nix and Windows
-            /angular[\\\/]core[\\\/]@angular/,
+            /**
+             * Override the initial configuration for ContextReplacementPlugin.
+             * The argument /angular[\\\/]core[\\\/]@angular/ causes several warnings.
+             * See https://prnt.sc/lezevh
+             *
+             * It's necessary to remove the @angular part and fix the issue by enabling
+             * the proper module.rules.Rule for @angular/core/*.js files(see above).
+             */
+            /angular[\\\/]core/,
             path.join(builder.require.cwd, 'src'),
```

Comment:
Do we really need to append `src` here? Could you check Svyat? I think we don't need to make assumption that all the package source code will be inside `src` and everything will work even if we don't append `src` here

Reply:
As far as I am concerned, we can remove the last two arguments in `webpack.ContextReplacementPlugin` and just pass the correct regexp `angular[\\\/]core`. 
Tested the project with `webpack.ContextReplacementPlugin(/angular[\\\/]core/)` for the Angular ^4, ^5, ^6, ^7 versions.